vedro_allure_reporter/_allure_reporter.py

```diff
@@ -69,14 +88,25 @@ def on_arg_parse(self, event: ArgParseEvent) -> None:
                            default=self._attach_scope,
                            help="Attach scope to Allure report")
 
-    def on_arg_parsed(self, event: ArgParsedEvent) -> None:
+    def on_subscribe_arg_parse(self, event: ArgParseEvent) -> None:
+        group = event.arg_parser.add_argument_group("Allure Reporter")
+        group.add_argument("--allure-labels",
+                           default=[],
+                           nargs="*",
```

Comment:
I believe it would be better to require at least one label as mandatory (`nargs="+"`)